# Description: Fix "always_inline function might not be inlinable" warnings, by
#  using "inline" with the "always_inline" attribute
# Author: Chris Coulson <chris.coulson@canonical.com>

diff --git a/third_party/libjpeg_turbo/config.h b/third_party/libjpeg_turbo/config.h
--- a/third_party/libjpeg_turbo/config.h
+++ b/third_party/libjpeg_turbo/config.h
@@ -64,17 +64,17 @@
 #define HAVE_UNSIGNED_SHORT 1
 
 /* Compiler does not support pointers to undefined structures. */
 /* #undef INCOMPLETE_TYPES_BROKEN */
 
 /* How to obtain function inlining. */
 #ifndef INLINE
 #if defined(__GNUC__)
-#define INLINE __attribute__((always_inline))
+#define INLINE inline __attribute__((always_inline))
 #elif defined(_MSC_VER)
 #define INLINE __forceinline
 #else
 #define INLINE
 #endif
 #endif
 
 /* libjpeg API version */
